numjuggler/utils/io.py

```diff
@@ -0,0 +1,13 @@
+import os
```

Comment:
Modules ``io.py`` and ``resource.py`` are used only for testing and seem to provide general functionality. If they are not numjuggler-specific, it is better to put them into a separate repository/package so they could be used e.g. in other repositories.

Reply:
These are just two files. Add repository just for them? Besides, to my experience latter these files become more specific for the project. These are just initial seeds.

Reply:
Just recently I've worked with likefunction code and added tests for it. To do that, I've extended a bit and moved some common code to the io.py . So, it's growing and, certainly, will grow further.
These changes will appear with the next change request